b2sdk/sync/file.py

```diff
@@ -109,7 +109,7 @@ def size(self):
         pass
 
 
-class LocalFileVersion(AbstractFileVersion):
+class LocalSyncFileVersion(AbstractFileVersion):
     """
     Hold information about one version of a file.
```

Comment:
adjust docstring for sync. Why sync?

Reply:
The `Sync` infix is here, because without it, we would have `AbstractFileVersion` and that would look like a base class for `FileVersion`, while it's not. These classes are only used in sync, therefore the infix.